src/parser/open-api-v3/open-api-v3-parser.js

```diff
@@ -2,54 +2,113 @@ import refParser from 'json-schema-ref-parser';
 import getUIReadySchema from './schemaParser';
 
 /**
+ * Sort function
+ *
+ * @param {String} str1
+ * @param {String} str2
+ *
+ * @return {number}
+ */
+function sortByAlphabet(str1, str2) {
+  if (str1 < str2) {
+    return -1;
+  } else if (str1 > str2) {
+    return 1;
+  } else {
+    return 0;
+  }
+}
+
+/**
+ * Sort function
+ *
+ * @param {String} inMethod1
+ * @param {String} inMethod2
+ *
+ * @return {number}
+ */
+
+function defaultMethodSort(inMethod1, inMethod2) {
+  const methodWeights = {
+    GET: 1,
+    POST: 2,
+    PUT: 3,
+    DELETE: 4,
+    HEAD: 5,
+    OPTIONS: 6,
+    TRACE: 7,
+    CONNECT: 8
+  };
+
+  const method1 = inMethod1.toUpperCase();
+  const method2 = inMethod2.toUpperCase();
+
+  if (!methodWeights.hasOwnProperty(method1)) {
```

Comment:
The check to filter out bad methods needs to be done somewhere (to be added). This extra check is only for the sort function to protect itself, i.e. make sure it only deals with valid data.